src/foundry/common/primitives/string.d.mts

```diff
@@ -38,6 +38,12 @@ declare global {
        * @defaultValue `false`
        */
       strict?: boolean;
+
+      /**
+       * Lowercase the string.
+       * @defaultValue `true`
+       */
+      lowercase?: boolean;
```

Comment:
You should probably add `| undefined` here. Could you make that change for all of `SlugifyOptions`? I know we aren't consistent about this but generally speaking Foundry allows explicit `undefined`s not just the implicit ones from leaving the property out of the object entirely.